dicom/config_service.go

```diff
@@ -0,0 +1,84 @@
+package dicom
```

Comment:
This file seems like a "base class", it contains the struct definitions for other (but not all) api's, but also contains the SetCrdServiceAccount. I would expect a config_service_cdr_account file, only containing these methods. I would also just move all struct definitions to their usage file.

Reply:
The `dicom` package is the first one where I pervasively grouped structs and member functions in separate files. I was a bit sloppy here and that should be fixed in 4907c0e . By splitting it up it also uncovered missing test code for the `SetCDRServiceAccount` code path, thanks! Appreciate the very thorough code review 👍🏻